### PR TITLE
document ms graph ingester

### PR DESCRIPTION
**Pull request by floren**

### CODE REVIEW

File: ingesters/ingesters.md [hunk 1]
```diff
@@ -1273,6 +1273,54 @@ Tenant-Domain=mycorp.onmicrosoft.com
 	Tag-Name="365-dlp"
 ```
 
+## Microsoft Graph API Ingester
+
+Gravwell provides an ingester which can pull security information from Microsoft's Graph API. In order to configure the ingester, you will need to register a new *application* within the Azure Active Directory management portal; this will generate a set of keys which can be used to access the logs. You will need the following information:
+
+* Client ID: a UUID generated for your application via the Azure management console
```
Comment (djfritz) [inline ingesters/ingesters.md:1280]: Mixed case: "a UUID", "a secret", "The domain". Probably make a->A
```suggestion
* Client ID: A UUID generated for your application via the Azure management console
```

File: ingesters/ingesters.md [hunk 2]
```diff
@@ -1273,6 +1273,54 @@ Tenant-Domain=mycorp.onmicrosoft.com
...
+* Client secret: a secret token generated for your application via the Azure console
```
Comment (djfritz) [inline ingesters/ingesters.md:1281]:
```suggestion
* Client secret: A secret token generated for your application via the Azure console
```

**Review by djfritz — CHANGES_REQUESTED** [review]

**Comment by floren** [comment]
ptal

**Review by djfritz — APPROVED** [review]
LGTM

**Pull request merged**